code/modules/mining/machine_vending.dm

```diff
@@ -12,6 +12,7 @@
 	var/icon_deny = "mining-deny"
 	var/obj/item/card/id/inserted_id
 	var/list/prize_list = list( //if you add something to this, please, for the love of god, sort it by price/type. use tabs and not spaces.
+		new /datum/data/mining_equipment("Necropolis Chest Key",		/obj/item/chest_key,												2000),
```

Comment:
probably shouldn't be first, between the advanced healing kit and fulton beacon would make most sense to me since its between the generally useful stuff and the mining specific stuff

Reply:
changed